.github/container/Dockerfile.pax.arm64

```diff
@@ -24,39 +22,35 @@ RUN wget https://github.com/bazelbuild/bazelisk/releases/download/v1.17.0/bazeli
 #------------------------------------------------------------------------------
 
 FROM wheel-builder as tftext-builder
-
-RUN <<"EOT" bash -exu
-set -o pipefail
+ARG SRC_PATH_TFTEXT
+RUN <<"EOF" bash -exu -o pipefail
 pip install tensorflow_datasets==4.9.2 auditwheel tensorflow==2.13.0
-git clone http://github.com/tensorflow/text.git /opt/tensorflow-text
-cd /opt/tensorflow-text
-git checkout v2.13.0
+get-source.sh -l tensorflow-text -m ${MANIFEST_FILE}
+cd ${SRC_PATH_TFTEXT}
 ./oss_scripts/run_build.sh
-EOT
+EOF
 
 #------------------------------------------------------------------------------
 # build lingvo
 #------------------------------------------------------------------------------
 
 FROM wheel-builder as lingvo-builder
-ARG REPO_LINGVO=https://github.com/tensorflow/lingvo.git
-ARG REF_LINGVO=master
-ARG SRC_PATH_LINGVO=/opt/lingvo
+ARG SRC_PATH_TFTEXT
+ARG SRC_PATH_LINGVO
 
-COPY --from=tftext-builder /opt/tensorflow-text/tensorflow_text*.whl /opt/
+COPY --from=tftext-builder ${SRC_PATH_TFTEXT}/tensorflow_text*.whl /opt/
 
-RUN get-source.sh -f ${REPO_LINGVO} -r ${REF_LINGVO} -d ${SRC_PATH_LINGVO}
+RUN get-source.sh -l lingvo -m ${MANIFEST_FILE}
 
 # build lingvo
-RUN <<"EOT" bash -exu
-set -o pipefail
+RUN <<"EOF" bash -exu -o pipefail
```

Comment:
I mean we don't have to rename, but I feel that it's more clear since a question already came up about its significance. This way the identifier is also self-documenting